lib/galaxy_test/api/test_configuration.py

```diff
@@ -23,6 +24,11 @@
 
 class ConfigurationApiTestCase(ApiTestCase):
 
+    def test_whoami(self):
+        response = self._get("whoami")
+        self._assert_status_code_is(response, 200)
+        assert response.json()['email'] == DEFAULT_TEST_USER
```

Comment:
This will pass on CI but isn't correct - maybe change this to TEST_USER not DEFAULT_TEST_USER at least.

Reply:
Fixed, thanks!